### ISSUE TITLE
Is there a developer fee for this service?

### ISSUE THREAD

**Issue by junhoi-pk** [EvgeniyKorepov/TonMinerPro#2]
Is there a developer fee for this service?

**Comment by EvgeniyKorepov** [comment]
Yes, on the statistics website this information is published at the top https://korepov.com/ton/minerpro/
**The application has devfee 1% (one share out of a hundred goes to the developer)**

At the same time, mining is not interrupted, unlike my Watchdog4DanilaMiner (where about 30 seconds are lost for restarts)

P.S. I added this information to the github page, thanks for noticing its absence.
